src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -144,6 +148,17 @@
      */
     public static final Notify DEFAULT_NOTIFICATION_LEVEL = Notify.ALL;
 
+    private transient static final String gerritCmdBuildStartedDefaultValue = "gerrit review <CHANGE>,<PATCHSET> --message 'Build Started <BUILDURL> <STARTED_STATS>' "
```

Comment:
According to [StackOverflow](https://stackoverflow.com/questions/35083970/what-is-the-effect-of-static-final-transient-in-java/35084120) static fields are by default transient so no need to declare them `transient`.
Also Java constant naming convention is UPPERCASE.

Reply:
Done & done.